sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample_GetAndListOperations.md

```diff
@@ -0,0 +1,77 @@
+# Get and List Document Model Operations
+
+This sample demonstrates how to Get and List all document model operations (succeeded, in-progress, failed) associated with the Form Recognizer resource. Note that operation information only persists for 24 hours.
+If the operation was successful, the document model can be accessed using `GetModel()` or `GetModels()` APIs.
+
+To get started you'll need a Cognitive Services resource or a Form Recognizer resource.  See [README][README] for prerequisites and instructions.
+
+## Creating a `DocumentModelAdministrationClient`
+
+To create a new `DocumentModelAdministrationClient` you need the endpoint and credentials from your resource. In the sample below you'll use a Form Recognizer API key credential by creating an `AzureKeyCredential` object, that if needed, will allow you to update the API key without creating a new client.
+
+You can set `endpoint` and `apiKey` based on an environment variable, a configuration setting, or any way that works for your application.
+
+```C# Snippet:CreateDocumentModelAdministrationClient
+string endpoint = "<endpoint>";
+string apiKey = "<apiKey>";
+var credential = new AzureKeyCredential(apiKey);
+var client = new DocumentModelAdministrationClient(new Uri(endpoint), credential);
+```
+
+## Get and List Document Model Operations
+
+Note that operation information only persists for 24 hours. If the operation was successful, the document model Id is provided in the `Result` property. This Id can be used with
+other methods like for example, `GetModel()`.
```

Comment:
seems like weird newline, idk

Reply:
this is for editor purposes. See how it doesn't affect the end .md file => https://github.com/Azure/azure-sdk-for-net/blob/feature/formrecognizerv4/sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample_GetAndListOperations.md